public/css/style.css

```diff
@@ -307,9 +307,7 @@
 }
 
 .btn-align {
```

Comment:
This class is now not needed, as justify-content has no effect unless display is set to flex